script/install-bundles/src/lib.rs

```diff
@@ -0,0 +1,95 @@
+use std::path::PathBuf;
+
+use hc_sandbox::calls::ActivateApp;
+use hc_sandbox::expect_match;
+use hc_sandbox::CmdRunner;
+use holochain_cli_sandbox as hc_sandbox;
+use holochain_conductor_api::AdminRequest;
+use holochain_conductor_api::AdminResponse;
+use holochain_conductor_api::conductor::ConductorConfig;
+use holochain_types::prelude::AppBundleSource;
+use holochain_types::prelude::InstallAppBundlePayload;
+use std::path::Path;
+
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Input {
+    #[structopt(short, long, default_value = "holochain")]
+    holochain_path: PathBuf,
+    happ: Option<PathBuf>,
+}
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    // Get and parse any input.
+    let input = Input::from_args();
+
+    println!("Starting installation process");
+
+    // Using the default mem network.
+    // let network = KitsuneP2pConfig::default();
+
+    // Create a conductor config.
+    let hc_dir =  PathBuf::from(r"./");
+    let config = ConductorConfig::load_yaml(Path::new("./config.yaml"))?;
+    println!("Generating sandbox..");
+    let path = hc_sandbox::generate::generate_with_config(Some(config), Some(hc_dir.clone()), Some(PathBuf::from(r".sandbox")))?;
```

Comment:
No need to use a raw string

```suggestion
    let path = hc_sandbox::generate::generate_with_config(Some(config), Some(hc_dir.clone()), Some(PathBuf::from(".sandbox")))?;
```